alf/algorithms/ppo_loss.py

```diff
@@ -52,9 +54,9 @@ def __init__(self,
          + td_loss_weight * td_loss (L^{VF} in equation (9))
          - entropy_regularization * entropy)
 
-        Note: There is a difference with baseline.ppo2 implementation. Here the
+        Note: There is a difference with baselines.ppo2 implementation. Here the
```

Comment:
Do we still want to keep this PPO loss? It might be misleading to people that this corresponds to PPO1.

Reply:
removed